### PR TITLE
FRW-172 Upgrade to Codeception 5

### PR DESCRIPTION
**Pull request by geega**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/FRW-172

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         "spryker/symfony": "^3.5.0"
     },
     "require-dev": {
-        "codeception/module-asserts": "^1.3.0",
+        "codeception/module-asserts": "^1.3.0 || ^3.0.0",
```
Comment (asmarovydlo) [inline composer.json:14]: Why do we need to support both versions?

Comment (geega) [reply composer.json:14]: for compatibility Codeception version 4 and 5
https://github.com/Codeception/module-asserts/blob/3.0.0/composer.json#L30

Comment (asmarovydlo) [reply composer.json:14]: We keep codeception 5 support only, please drop 1.3.0 constraint.

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

**Comment by geega** [comment]
Reverted composer.json for support Codeception 4 and PHP 7.4 
Also CI validateion 8.1 will be fixed after release new module testify

**Review by asmarovydlo — APPROVED** [review]

**Comment by dereuromark** [comment]
Merge only was invalid due to composer changes!

**Pull request merged**